### ISSUE TITLE
Redis Password Option

### ISSUE THREAD

**Issue by ljades** [opencv/cvat#3133]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour
Environment variables should be available to configure the Redis deploy with a password, and to incorporate said password into the RQ, Cacheops and RQ scheduler configurations in the cvat server.

### Current Behaviour
No option to use passwords.

### Possible Solution
RQ, Cacheops and RQ scheduler all have built-in options for setting password already, so the server would need to simply add them to its supervisord.conf and production settings.

### Context
I'm trying to deploy an instance of CVAT in my company's kubernetes environment, and we require all our redis instances to have password components. Because there isn't even the option within the python code, we're going to need to custom-roll our solution by either forking the repo and running our own image builds, or mounting modified python settings files as configMaps. Neither of which is as clean as we'd like.

### Your Environment

- Using Docker images of version 1.3.0 for both server and UI

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

**Comment by azhavoro** [comment]
@ljades May I ask you to prepare a PR with your changes, or is this not possible for legal reasons or something else?

**Comment by ljades** [comment]
@azhavoro I can make a PR, sure!

Currently running into other issues, but once I know it works, PR it.

(Maybe this is deserving of another Issue, but the other issue I'm running into right now involves supervisord not working properly because almost all of the program workers keep erroring out with

```
bash: cannot set terminal process group (-1): Inappropriate ioctl for device
bash: no job control in this shell
```

the runserver worker also errors with:

```
Failed to import module 'tf_detection_api_format.converter.py': Can't import tensorflow. Test process exit code: -9.
```

This only happens on kubernetes runs, not docker-compose runs, and I'm trying to figure out why.)

**Comment by SpecLad** [comment]
This was implemented in #5505.
